Riot/Modules/Room/VoiceMessages/VoiceMessageAttachmentCacheManager.swift

```diff
@@ -275,4 +309,13 @@ class VoiceMessageAttachmentCacheManager {
         
         MXLog.debug("[VoiceMessageAttachmentCacheManager] Failed task with error: \(error)")
     }
+    
+    private func setupTemporaryFilesFolder() throws {
+        let url = temporaryFilesFolderURL()
+        try FileManager.default.createDirectory(at: url, withIntermediateDirectories: true, attributes: nil)
+    }
+    
+    private func temporaryFilesFolderURL() -> URL {
```

Comment:
nit: Could be a `var`?

Reply:
It could yes. Any particular reason you prefer that?

Reply:
Just a code style preference. Happy if to leave if people have a preference for this style.

Reply:
After a team chat decided to go with a property unless it takes parameters, mutates state or is heavy.